tripal/tripal_jbrowse
# Make package more cohesive: Step 1

Issue #25 

This PR 
1. Extends "Tripal JBrowse Management" to keep track of start location and initial tracks to display. 
2. Adds edit functionality for JBrowse instances created by "Tripal JBrowse Management"
3. Fixes some minor bugs: ensure module is configured, settings tab shows up, breadcrumbs are accurate.

Testing
1. Check that existing JBrowse instance functionality is not altered (launch link is unchanged)
2. Check that you can edit existing JBrowse instances with or without entering start location and tracks
3. Check you can create a new JBrowse instance with or withour entering start location and tracks.
